src/services/item/plugins/app/appData/appData.schemas.ts

```diff
@@ -131,3 +131,38 @@ export const getForOne = {
     '4xx': errorSchemaRef,
   },
 } as const satisfies FastifySchema;
+
+export const upload = {
+  operationId: 'createAppDataFile',
+  tags: ['app', 'app-data', 'file'],
+  summary: 'Create app data file',
+  description: `Upload a file to create a corresponding app data. The created app data will have a type file and visibility ${AppDataVisibility.Member}. The data property will contain the file properties.`,
```

Comment:
Is it possible / do we want to allow a file to be uploaded as a Visibility.Item ?